ocaml/xapi/xapi_session.mli

```diff
@@ -103,3 +103,5 @@ val record_login_failure :
   -> 'a
 
 val get_failed_login_stats : unit -> string option
+
+val total_sessions : int Atomic.t
```

Comment:
For isolation purposes it'd be better to expose a function that cannot modify the value

```ocaml
  val get_total_sessions : unit -> Int64.t
  (** Retrieves the amount of sessions opened since the last time xapi was started *)

  let get_total_sessions () = Atomic.get total_sessions |> Int64.of_int
```